code/renderers/svelte/src/___test___/Button.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
```

Comment:
These following files have 3 underscores in the directory `/_ _ _test_ _ _/`, is that on purpose? I just did a search in the repo and we don't use that anywhere else.